clients/analysis/src/__tests__/__resources__/cfgraph/case29.diagnostics.json

```diff
@@ -1,69 +0,0 @@
-[
-  {
-    "range": {
-      "start": {
-        "line": 12,
-        "character": 0
-      },
-      "end": {
-        "line": 12,
-        "character": 20
-      }
-    },
-    "message": "Implicit EXIT PROGRAM statement executed",
```

Comment:
Shouldn't the implicit exit message still be generated for parag2?

Reply:
No, because now it is under the condition